### PR TITLE
7/reusable card component

### PR DESCRIPTION
**Pull request by jobkarani**
#28 feat: Creates a reusable and responsive card 

elewaweb
elewamobile

### CODE REVIEW

File: apps/elewa-group-website/src/app/app.module.ts
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/app/app.module.ts:9]: Use domain imports i.e 
import { CardsModule } from '@elewa-group/features/components/cards'

File: libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss:3]: Use variables in _variables.scss for colors, fonts etc

File: libs/features/components/cards/src/lib/features-components-cards.module.ts
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/features-components-cards.module.ts:7]: Rename to CardsModule

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Comment by IanOdhiambo9** [comment]
Use fonts in _variables.scss

**Comment by IanOdhiambo9** [comment]
Resolve conflict on styles.scss

File: apps/elewa-group-website/src/app/app.module.ts
Comment (jobkarani) [reply apps/elewa-group-website/src/app/app.module.ts]: Resolved the issue. Waiting for another review.

File: libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss
Comment (jobkarani) [reply libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss]: Resolved the issue. Waiting for another review.

File: libs/features/components/cards/src/lib/features-components-cards.module.ts
Comment (jobkarani) [reply libs/features/components/cards/src/lib/features-components-cards.module.ts]: Resolved the issue. Waiting for another review.

**Review by IanOdhiambo9 — APPROVED** [review]

**Comment by jrosseel** [comment]
Closed bounty as WON with 3 bounty points scored

**Pull request merged**
